.github/workflows/client-verify.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Install tox
         run: |
-          pip install tox==4.16.0
+          pip install git+https://github.com/tox-dev/tox.git@2f7c60ec71d47f4099394af11999a840bc97c9a9 #tox==4.17.0
```

Comment:
Why are we installing it from git directly, Aki? I think I would prefer the other approach

Reply:
Basically it's my comment in the rest of the places where we are using git

Reply:
That's the way to install using the hash value by pip.  The scan tool doesn't like the version nunber.

Reply:
I think it may be worth an internal discussion on whether we need to implement all these recommendations

Reply:
Agree, I don't know what best practice or security problem are we trying to cover with this to be honest 😅